e_chil.c

```diff
@@ -1287,17 +1554,16 @@ static void hwcrhk_log_message(void *logstr, const char *message)
 {
     BIO *lstream = NULL;
 
-    CRYPTO_w_lock(CRYPTO_LOCK_BIO);
     if (logstr)
         lstream = *(BIO **)logstr;
     if (lstream) {
         BIO_printf(lstream, "%s\n", message);
     }
-    CRYPTO_w_unlock(CRYPTO_LOCK_BIO);
 }
 
 /*
- * This stuff is needed to compile this ENGINE into a DSO.
+ * This stuff is needed if this ENGINE is being compiled into a
```

Comment:
If I may be so picky, not "if".  We *are* building a DSO ("dlopenable" in GNU terms), there's no alternative here.

Reply:
Done